.github/workflows/release.yml

```diff
@@ -40,6 +40,42 @@ jobs:
             --no-git-commit \
             skip
 
+  npm_publish:
+    name: Publish NPM package (wasm)
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: wasm32-unknown-unknown
+          profile: minimal
+          override: true
```

Comment:
Would prefer if we used Dtolnay's action, since actions-rs repos are unmaintained.

Reply:
I just noticed we are using the actions-rs actions everywhere in our codebase. Should we do this change separately, to migrate to the dtolnay's action?

Reply:
Yep, that would be preferable.